docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -1,22 +1,60 @@
 ---
 title: Joining Nodes in AWS
-description: How to join nodes and proxies on EC2
-h1: Joining Node and Proxies in AWS
+description: How to join nodes and proxies running on AWS
+h1: Joining Nodes and Proxies in AWS
 ---
 
-This section will explain how to set up Teleport Nodes and Proxies to join your
-Teleport cluster with the EC2 join method. The EC2 join method enables you to
-allow Teleport nodes to join your Teleport cluster from EC2 instances in your
-AWS account without sharing any secrets.
+This guide will explain how to set up Teleport Nodes and Proxies to join your
+Teleport cluster without sharing any secrets when they are running in AWS.
+
+<Admonition type="note">
+You can also configure Nodes running in AWS to join a cluster via secret tokens,
+which is useful when you don't want to rely on AWS-specific APIs.
+Read more in the following guide:
+[Getting Started](../../getting-started/linux-server.mdx#add-a-node-to-the-cluster)
```

Comment:
Should we use our dedicated page on adding nodes instead of a section in the getting started guide? https://goteleport.com/docs/setup/admin/adding-nodes/

Reply:
yep, that looks like a much betting link, updated